src/include/clSPARSE.h

```diff
@@ -464,6 +464,15 @@ CLSPARSE_EXPORT clsparseStatus
 clsparseDdense2csr(const cldenseMatrix* A, clsparseCsrMatrix* csr,
                    const clsparseControl control);
 
+//SpM x SpM
+ CLSPARSE_EXPORT clsparseStatus
+        clsparseScsrSpGemm(
+        const clsparseCsrMatrix* sparseMatA,
+        const clsparseCsrMatrix* sparseMatB,
+              clsparseCsrMatrix* sparseMatC,
+        const clsparseControl control );
+
+
```

Comment:
Related to Jakubs comments below, how much work would it be to add 'D' precision?  Can you do it now, or do you prefer to wait for a separate pull request?